lib/backend/repository/repository.go

```diff
@@ -302,7 +302,7 @@ func getLayer(imgID, registry string, repoData *RepoData, imgSize int64, tmpDir
 	progressReader := &ioprogress.Reader{
 		Reader:       res.Body,
 		Size:         imgSize,
-		DrawFunc:     ioprogress.DrawTerminalf(os.Stdout, fmtfunc),
+		DrawFunc:     ioprogress.DrawTerminalf(os.Stderr, fmtfunc),
```

Comment:
What I mean is that users of docker2aci should not have it printing to stderr unless they request it